docs/source/gettingstarted.rst

```diff
@@ -68,6 +68,29 @@ There is an open issue to support logging in without any third-party platform: `
 To log in to PolicyKit, you'll need to install it on a server and set up at least 1 of the auth-enabled integrations.
 
 
+Local Development (with docker)
+-----------------
+
+You can also deploy PolicyKit using Docker.
+
+1. Make sure to have Docker and Docker Compose installed.
+
+2. Clone the repo, navigate to the root and create an ``.env`` file as on the previous guide:
```

Comment:
Would it be possible to have it work so that you didn't have to create a custom `.env` file for Docker but that the variables would automatically be set to pull in the DB and Rabbit connections automatically? I could see this being helpful as we extend the docker compose file with any more service as well, then they should just work without requiring anything on the users part.

Like could be done by adding the env variables directly to the web service in the compose file and [extending it](https://docs.docker.com/compose/multiple-compose-files/extends/#extending-services-within-the-same-file-and-from-another-file) for the celery services, or use a special `.env.docker` file that is pulled into those services?

Reply:
Sorry I missed the last comment on yesterday where you also pointed this.

> then they should just work without requiring anything on the users part.

I didn't solve this in the exact way you suggested, but I think we are getting the same effect anyway. I removed the DB and Rabbit config from `.env.example` since we now have defaults on `settings.py` and added them as environment variables to `docker-compose.yml`. This way we have correct defaults when running both on docker and without it.

Reply:
Thank you!